### ISSUE TITLE
After clicking Reload button, predefined cropping ratio is lost

### ISSUE THREAD

**Issue by walkowl** [scaleflex/filerobot-image-editor#177]
Config:
```
      Crop: {
        ratio: 147 / 192,
        noPresets: true
      },
```
Actions:
1) Click "Reload" button (the one next to Save)
2) Confirm the reload
3) Ratio is lost

**Comment by AhmeeedMostafa** [comment]
@walkowl thanks for your report, we'll consider the fix for this issue in the next release

**Comment by AhmeeedMostafa** [comment]
@walkowl the bug is fixed in the new release (v4.2.0) please check and let us know

**Comment by walkowl** [comment]
Thanks @AhmeeedMostafa Works as expected 👍
